src/main/java/jdbchomework/service/CompanyService.java

```diff
@@ -1,37 +1,54 @@
 package jdbchomework.service;
 
+import jdbchomework.dao.hibernate.CompaniesHibDao;
 import jdbchomework.dao.model.CompaniesDao;
 import jdbchomework.entity.Company;
+import jdbchomework.entity.Developer;
 
 import java.util.List;
 
 public class CompanyService {
 
     private CompaniesDao companiesDao;
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Remove this unused "companiesDao" private field. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS1068)